charts/victoria-metrics-single/templates/_helpers.tpl

```diff
@@ -213,3 +213,12 @@ Return license volume mount for container
   readOnly: true
 {{- end -}}
 {{- end -}}
+
+{{/*
+Enforce license for vmbackupmanager
+*/}}
+{{- define "victoria-metrics.vmbackupmanager.enforce_license" -}}
+{{ if and .Values.server.vmbackupmanager.enable (not (or .Values.server.vmbackupmanager.eula .Values.license.key .Values.license.secret.name)) }}
+{{ fail "Pass -eula command-line flag or valid license at .Values.license if you have an enterprise license for running this software. See https://victoriametrics.com/legal/esa/ for details"}}
```

Comment:
can we add information about docs, ent page and trial here?

Reply:
sure, added